metrics/service.go

```diff
@@ -0,0 +1,59 @@
+package metrics
+
+import (
+	"github.com/mysteriumnetwork/openvpn-forwarder/proxy"
+	"github.com/prometheus/client_golang/prometheus"
+	"time"
+)
+
+type Service struct {
+	proxyRequestDuration *prometheus.HistogramVec
+	proxyRequestData     *prometheus.CounterVec
```

Comment:
Would be nice to add also proxyRequests counter to see a number of requests going through OpenVPN forwarder.

Reply:
Added

Reply:
It's nice to have also `proxyNumberOfConnecions`, but I was asking about the number of requests always growing counter. 
It might be useful if we want to see how many requests are going through the OpenVPN Forwarder at some time.
it also can show number of requests we skipped(which is going directly) and processes by our service.

Reply:
Added two more metrics